src/browser/components/Alert.react.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+if (process.env.IS_BROWSER) {
```

Comment:
Do we expect this to ever be `false`?

Reply:
Good point. @k4wo what do you think about it?

Reply:
I don't know a background but it's for server rendering.